package exception;


public class Main {
	
	public static void main(String[] args) {
		
		
		
		System.out.println("1");
		System.out.println("1");
		System.out.println("!");
		System.out.println("1");
		System.out.println("1");
		try {
			
			int a = 10/0;
			
		} catch (Exception e) {
			System.out.println(e);
		}
	
		finally {
			System.out.println();
			
		}
			
	}

}
